scylla-cdc/src/cdc_types.rs

```diff
@@ -40,3 +40,13 @@ impl StreamID {
         StreamID { id: stream_id }
     }
 }
+
+pub trait ToTimestamp {
```

Comment:
I don't see it being used in any public interface, so let's keep it `pub(crate)` for now so that users of the library won't depend on it.

Reply:
Well, printer needs it to be public.

Reply:
That's not a good reason to keep it public. This is a helper trait for internal purposes of both the library and the replicator, and the library's interface does not need it. We don't want to support it.

While not ideal, I'd recommend just introducing a duplicated definition of the trait in the replicator.